{"name": "Тест ввода почты и нажатия на кнопку восстановить", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_password_recovery.TestPasswordRecovery object at 0x10294f740>\n\n    @allure.title(\"Тест ввода почты и нажатия на кнопку восстановить\")\n    @pytest.mark.usefixtures(\"driver\")\n    def test_is_active_input_password(self):\n        self.driver.get(main_url)\n        client = PagePasswordRecovery(self.driver)\n>       result = client.check_focused_input_password_recovery()\n\ntests/test_password_recovery.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages/page_password_recovery.py:43: in check_focused_input_password_recovery\n    self.click_virt_mouse(self.button_recovery_password)\npages/base_page.py:53: in click_virt_mouse\n    WebDriverWait(self.driver, 5).until(EC.element_to_be_clickable(element))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e0606e6c-95d7-420b-8376-2334002627f2\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x102a17240>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nvenv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "steps": [{"name": "Проверка активности поля ввода пароля", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/pages/page_password_recovery.py\", line 43, in check_focused_input_password_recovery\n    self.click_virt_mouse(self.button_recovery_password)\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/pages/base_page.py\", line 53, in click_virt_mouse\n    WebDriverWait(self.driver, 5).until(EC.element_to_be_clickable(element))\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Принудительный клик ['css selector', '.button_button__33qZ0']", "status": "passed", "parameters": [{"name": "element", "value": "['css selector', '.button_button__33qZ0']"}], "start": 1726779417775, "stop": 1726779418052}, {"name": "Принудительный клик ['xpath', '/html/body/div/div/main/div/div/p[2]/a']", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/pages/base_page.py\", line 53, in click_virt_mouse\n    WebDriverWait(self.driver, 5).until(EC.element_to_be_clickable(element))\n  File \"/Users/ulabualskaa/Documents/GitHub/Diplom_3/venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "element", "value": "['xpath', '/html/body/div/div/main/div/div/p[2]/a']"}], "start": 1726779418053, "stop": 1726779423139}], "start": 1726779417775, "stop": 1726779423140}], "attachments": [{"name": "log", "source": "9fb3fc9c-3891-475b-87bd-31a34a78a6ee-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1726779417589, "stop": 1726779423142, "uuid": "c7cf743f-86d5-4ccd-b1eb-42789c1179fb", "historyId": "f829491e0b698c0d923458d45e32ac8b", "testCaseId": "4da1eb2b32829e2f56e9884ea034844b", "fullName": "tests.test_password_recovery.TestPasswordRecovery#test_is_active_input_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_password_recovery"}, {"name": "subSuite", "value": "TestPasswordRecovery"}, {"name": "host", "value": "MacBook-Pro-Ula.local"}, {"name": "thread", "value": "14340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_password_recovery"}]}